### PR TITLE
Feat(TokenScript docs): Added a TokenScript documentation page

### LINKED ISSUE #1: TokenScript Docs
Docs needed for [TokenScript Agent](https://w3gpt.ai/?a=asst_13kX3wWTUa7Gz9jvFOqnnA77).

Tokenscript info:
https://www.tokenscript.org/

Demo video:
https://www.youtube.com/watch?v=DvdZp-DLiMw&t=1s

Can be added to [Features](https://github.com/Markeljan/web3gpt-docs/tree/main/content/features) or under a new folder `content/Agents`

Make sure to update `_meta.js`

### PR DESCRIPTION
**Pull request by paschal533**
## Description

This pull request adds a comprehensive documentation page for TokenScript Agent, aimed at helping users understand the process of creating, editing, and deploying smart contracts with AI assistance. The documentation covers the following key areas:

- Overview of TokenScript Agent and its core functionality.
- Step-by-Step Instructions for generating, reviewing, and deploying smart contracts using natural language prompts.
- Example Walkthroughs demonstrating basic (ERC20, ERC721) and advanced contract creation.
- Error Handling guidance for common issues users may encounter.

## Related Issues

fix #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "@radix-ui/react-slot": "^1.1.0",
     "@vercel/analytics": "^1.3.2",
     "class-variance-authority": "^0.7.0",
+    "mermaid": "^11.4.0",
```
Comment (Markeljan) [inline package.json]: Is this being used?

File: package-lock.json
Comment (Markeljan) [inline package-lock.json:1]: please use bun `curl -fsSL https://bun.sh/install | bash` or omit `package-lock.json` from commits

File: content/agents/tokenscript-agent.mdx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+```plaintext
+Deploy to Sepolia
+```
+And you will get this response from Our AI agent:
+
+
+The ERC721 NFT contract has been successfully deployed on the Sepolia network!
+
+- Contract Address: [0x1D449aA762E5F0E6E35E0663226445b9ce11942a](https://eth-sepolia.blockscout.com/address/0x1D449aA762E5F0E6E35E0663226445b9ce11942a)
+- IPFS Repository: [Source Code on IPFS](https://web3gpt.mypinata.cloud/ipfs/bafybeihc2foi6whsohrj6bsqkw5hn67agakoupu5goozyqjkozh2gassoy)
```
Comment (Markeljan) [inline content/agents/tokenscript-agent.mdx]:
```suggestion
- IPFS Repository: [Source Code on IPFS](https://ipfs.w3gpt.ai/ipfs/bafybeihc2foi6whsohrj6bsqkw5hn67agakoupu5goozyqjkozh2gassoy)
```

File: content/agents/tokenscript-agent.mdx [hunk 3]
```diff
@@ -0,0 +1,167 @@
+import { Bleed, Callout, Steps } from "nextra/components";
+
+# TokenScript Agent
+
+## Overview
+
+**TokenScript Agent** empowers users to create, deploy, and manage smart contracts seamlessly by leveraging Web3GPT AI-driven code generation. Users can simply describe their requirements, and TokenScript Agent will produce the Solidity code, suggest edits, and even handle deployment.
```
Comment (Markeljan) [inline content/agents/tokenscript-agent.mdx]:
```suggestion
**[TokenScript Agent](https://www.tokenscript.org)** empowers users to create, deploy, and manage smart contracts seamlessly by leveraging Web3GPT AI-driven code generation. Users can simply describe their requirements, and TokenScript Agent will produce the Solidity code, suggest edits, and even handle deployment.
```

**Review by Markeljan — CHANGES_REQUESTED** [review]
Lets goooo! Thanks for contributing @paschal533  requesting a few changes.

File: package.json (hunk 1 above)
Comment (paschal533) [reply package.json]: I got some errors when I tried to run `npm run dev` without it.

File: package-lock.json
Comment (paschal533) [reply package-lock.json:1]: Alright.

File: package.json (hunk 1 above)
Comment (Markeljan) [reply package.json]: @paschal533 I tested locally and works without `mermaid` using bun, removing for now

**Review by Markeljan — APPROVED** [review]
TY @paschal533 for your contribution and for being the first contributor to the docs!! 🎊 🥳

**Pull request merged**